android/anc/src/main/java/org/smartregister/fhircore/anc/ui/reports/ReportDataProvider.kt

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
We can transfer this class implementation to `anc/data/reports` package instead of `anc/ui/reports`

Reply:
class transferred.